pager/src/main/java/dev/chrisbanes/accompanist/pager/Pager.kt

```diff
@@ -53,12 +53,12 @@ import kotlin.math.roundToInt
  * The scroll threshold for moving to the next page. The value is used in both directions
  * (so both negative and positive).
  */
-internal const val ScrollThreshold = 0.4f
+internal const val ScrollThreshold = 0.35f
```

Comment:
Should this be configurable?

Reply:
Possibly? I'll add it to my follow-up list